modules/apps/web-experience/export-import/export-import-api/src/main/java/com/liferay/exportimport/staged/model/repository/StagedModelRepository.java

```diff
@@ -49,16 +84,32 @@ public default T fetchMissingReference(String uuid, long groupId) {
 	public List<T> fetchStagedModelsByUuidAndCompanyId(
 		String uuid, long companyId);
 
+	/**
```

Comment:
@akosthurzo can you add what this particular method (and the other methods that you've done this way) does? You redirect to other methods that developers should use, but we don't have a good understanding of what this interface provides for those methods. We can keep the recommended methods there, but just state them after the initial description of the method.